xcode/Subconscious/Shared/Components/AppView.swift

```diff
@@ -77,6 +77,7 @@ struct AppView: View {
             )
         }
         .onAppear {
+            SentryIntegration.start()
```

Comment:
We can reduce time to first paint by defering the SentryIntegration call, but we still want to catch any errors that are thrown via `AppAction.appear`

Reply:
We could defer everything that's happening in appear. For example, we could queue an fx in appear for an action called "start", and move everything downstream of that *except* the Sentry setup. This should cause the downstream actions to run on the next tick.

Noting. Would want to defer to a future PR. Worth profiling if there's a difference as well.